### PR TITLE
Ms 4659 use new openformat service

### PR DESCRIPTION
**Pull request by hwhadbc**
Request and response formats has changed significantly, so massive changes in the model layer.

### CODE REVIEW

File: src/test/java/dk/dbc/connector/openformat/OpenFormatConnectorTest.java [hunk 1]
```diff
@@ -56,127 +59,178 @@ static void stopWireMockServer() {
...
+        PromatElements elements = fetchAndAssertResponse("26485878", PromatElements.class);
 
-        assertThat("dk5", formatResponse.getPromat().get(0).getElements().getDk5(), is(notNullValue()));
-        assertThat("dk5 size", formatResponse.getPromat().get(0).getElements().getDk5().size(), is(2));
-        assertThat("dk5 0", formatResponse.getPromat().get(0).getElements().getDk5().get(0).getValue(), is("79.864"));
-        assertThat("dk5 1", formatResponse.getPromat().get(0).getElements().getDk5().get(1).getValue(), is("Olsen, Ole"));
+        assertThat("dk5", elements.getDk5(), is(notNullValue()));
+        assertThat("dk5 size", elements.getDk5().size(), is(2));
+        assertThat("dk5 0", elements.getDk5().get(0), is("79.864"));
+        assertThat("dk5 1", elements.getDk5().get(1).trim(), is("Olsen, Ole")); // Todo: There is currently an error in OpenFormat that puts a leading blank in front of " Olsen". When that is fixed, we can remove the 'trim()'
```
Comment (hwhadbc) [inline src/test/java/dk/dbc/connector/openformat/OpenFormatConnectorTest.java:227]: https://dbcjira.atlassian.net/browse/DEINBOX-11

**Review by atmdbc — APPROVED** [review]

File: src/main/java/dk/dbc/connector/openformat/model/OpenFormatResponse.java [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+                    }
+                }
+            }
+        }
+        return errors;
+    }
+
+    /**
+     * Helper method to dig down and get the first error
+     * @return error if found, empty list otherwise
```
Comment (MichelLyck) [inline src/main/java/dk/dbc/connector/openformat/model/OpenFormatResponse.java]: return null hvis ikke fundet, ikke en tom liste

File: src/test/resources/__files/body-format-GQHzJ.json [hunk 3]
```diff
@@ -0,0 +1 @@
+{"objects":[{"promat":[{"formatted":{"format":"promat","records":[{"elements":{"faust":["23319322"],"creator":["Samokovlija, Isak"],"dk5":["88.65"],"materialtypes":{"type":["Bog"]},"extent":["365 sider"],"publisher":["S.l., Novi Sad, 1972"],"title":["Pripovijetke"]}}]},"mediaType":"application/json"}]}],"trackingId":"abedf276-eda4-4115-8a85-3b9b41556aaa"}
```
Comment (MichelLyck) [inline src/test/resources/__files/body-format-GQHzJ.json:1]: Mangler du et hak i Settings → Editor → General → Ensure line feed at file end on save ?

**Review by MichelLyck — APPROVED** [review]

File: src/main/java/dk/dbc/connector/openformat/model/OpenFormatResponse.java (hunk 2 above)
Comment (hwhadbc) [reply src/main/java/dk/dbc/connector/openformat/model/OpenFormatResponse.java]: Done

**Pull request merged**